### ISSUE TITLE
No longer enabled in Chrome Canary

### ISSUE THREAD

**Issue by KibblesNBits** [YePpHa/YouTubeCenter#1827]
Has anyone addressed the issue that Google has now prevented Chrome Canary(developer version of Chrome) from accepting Youtube Center?  Is there a way to by-pass this.  I am able to load it, but when I restart the browser, Google kicks it out again. Any suggestions about how to fix?

Thanks

**Comment by mikhaelkh** [comment]
@KibblesNBits, install Opera or other browser based on Chromium, or if you like risk, you can install Chromium itself. More info [here](http://chromium.woolyss.com/).

**Comment by mikhaelkh** [comment]
Or you can install unpacked extension from local folder: go to chrome://extensions, enable developer mode, install unpacked extension. You can unpack *.crx as *.zip.

**Comment by Huskeyhund** [comment]
@mikhaelkh, thx a lot!

**Comment by The-MAZZTer** [comment]
Change has been pushed to Dev branch, Dev can no longer run YouTube Center as-is.

My workaround is to use the webstore ZIP file from dist/ in the GitHub. Extract and load as an unpacked extension (this is still permitted but Chrome will ask you to disable them every time it starts up).

**Comment by ErdeFB** [comment]
Chromium has never nagged me about unpacked extensions, so you guys might want to give that a try if that gets annoying.

**Comment by mikhaelkh** [comment]
@The-MAZZTer, OK, my bad, just use [7-zip](http://www.7-zip.org/download.html) or similar program, you don't need to rename *.crx

**Comment by The-MAZZTer** [comment]
@mikhaelkh I do use 7-zip actually and it couldn't read the CRX. I did not rename the file, I tried to use it as a ZIP is what I meant (I thought they were digitally signed ZIPs).

Anyway I just grabbed the webstore ZIP off of the latest git revision and used that. Side note: Blocking auto play seems to be broken in that version (videos can't start playing manually either).

@ErdeFB Odd because it nags me about mine every startup. Maybe it's just because it's Dev (Chromium nightlies are not intended for your regular browser so those warnings might be disabled).
